CHANGES.rst

```diff
@@ -5,7 +5,9 @@
 4.3.2 (unreleased)
```

Comment:
When dropping Python 3.4, @icemac recommended to bump the version to a new major number, as for instance Debian 8 comes with Python 3.4 and is still supported (until 2022-06).

Reply:
Thank you for reminding me.